tests/README.md

```diff
@@ -0,0 +1,83 @@
+
+Note that some tests require re-expansion and comparison, as list values may exist as values of properties that have `@container: @list` and the comparison algorithm will not consider ordering significant.
+
+# Running tests
+
+The top-level [manifest](manifest.jsonld) references the specific test manifests, which in turn reference each test associated with a particular type of behavior.
+
+Implementations create their own infrastructure for running the test suite. In particular, the following should be considered:
+
+* Test case properties identifying a file (_input_, _output_, _context_, _expectContext_, and _frame_) are presumed to have a media type appropriate for the file extension.
+  * `application/ld+json` for `.jsonld`
+  * `application/ld+yaml` for `.yamlld`
+  * `application/n-quads` for `.nq`
+* The media type for the file associated with the _input_ property can be overridden using the `contentType` option.
+* Some algorithms, particularly _fromRdf_, may not preserve the order of statements listed in the input document, and provision should be taken for performing unordered array comparison, for arrays other than values of `@list`. (This may be difficult for compacted results, where array value ordering is dependent on the associated term definition).
+* Some _toRdf_ tests require the use of [JSON Canonicalization Scheme](https://tools.ietf.org/html/draft-rundgren-json-canonicalization-scheme-05) to properly generate RDF Literals from JSON literal values. This algorithm is to be used to properly compare results using [RDF Dataset Isomorphism](https://www.w3.org/TR/rdf11-concepts/#dfn-dataset-isomorphism). These tests are marked using the `useJCS` option.
+* Some _toRdf_ tests may generate results as [Generalized RDF](https://www.w3.org/TR/rdf11-concepts/#section-generalized-rdf), specifically having a blank node predicate. Technically, these are in an invalid N-Quads format; provisions must be taken to be able to parse and recognize quads having a blank node predicate.
+* When comparing documents after flattening, framing or generating RDF, blank node identifiers may not be predictable. Implementations should take this into consideration. (One way to do this may be to reduce both results and _expected_ to datasets to extract a bijective mapping of blank node labels between the two datasets as described in [RDF Dataset Isomorphism](https://www.w3.org/TR/rdf11-concepts/#dfn-dataset-isomorphism)).
+* Some tests may have a `requires` property, indicating some optional behavior described by a test vocabulary term.
+
+# Building HTML Manifests
+
+The HTML versions of the test manifests are built using the Node action `npm run generate`, which also cross indexes tests with references from the specification.
+
+# Contributing
+
+If you would like to contribute a new test or a fix to an existing test,
+please follow these steps:
+
+1. Notify the JSON-LD mailing list, public-linked-json@w3.org,
```

Comment:
Though I do wonder, why not suggest they create an issue on the repo?
```suggestion
1. Notify the JSON-LD Working Group mailing list, [public-linked-json@w3.org](mailto:public-linked-json@w3.org),
```

Reply:
+1 to also add a reference to the github repo

Reply:
Check updated document, also in generated tests/*manifest.html files.